lib/Swagger/src/ContentEntity/Api/ContentFeedApi.php

```diff
@@ -99,6 +101,128 @@ public function setApiClient(\Swagger\Client\ApiClient $apiClient)
+    {
+        // verify the required parameter 'wiki_id' is set
+        if ($wiki_id === null) {
+            throw new \InvalidArgumentException('Missing the required parameter $wiki_id when calling getOrderedRelatedContent');
+        }
+        // parse inputs
+        $resourcePath = "/content-feed/from-wiki-id/{wikiId}";
+        $httpBody = '';
+        $queryParams = array();
+        $headerParams = array();
+        $formParams = array();
+        $_header_accept = $this->apiClient->selectHeaderAccept(array('application/json'));
+        if (!is_null($_header_accept)) {
+            $headerParams['Accept'] = $_header_accept;
+        }
+        $headerParams['Content-Type'] = $this->apiClient->selectHeaderContentType(array());
+
+        // query params
+        if ($limit !== null) {
+            $queryParams['limit'] = $this->apiClient->getSerializer()->toQueryValue($limit);
+        }
+        // query params
+        if ($include_root_relations !== null) {
+            $queryParams['includeRootRelations'] = $this->apiClient->getSerializer()->toQueryValue($include_root_relations);
+        }
+        // query params
+        if ($show_connection_details !== null) {
+            $queryParams['showConnectionDetails'] = $this->apiClient->getSerializer()->toQueryValue($show_connection_details);
+        }
+        // query params
+        if ($restrict_universe !== null) {
+            $queryParams['restrictUniverse'] = $this->apiClient->getSerializer()->toQueryValue($restrict_universe);
+        }
+        // query params
+        if (is_array($content_order)) {
+            $content_order = $this->apiClient->getSerializer()->serializeCollection($content_order, 'multi', true);
+        }
+        if ($content_order !== null) {
+            $queryParams['contentOrder'] = $this->apiClient->getSerializer()->toQueryValue($content_order);
+        }
+        // path params
+        if ($wiki_id !== null) {
+            $resourcePath = str_replace(
+                "{" . "wikiId" . "}",
+                $this->apiClient->getSerializer()->toPathValue($wiki_id),
+                $resourcePath
+            );
+        }
+        // default format to json
+        $resourcePath = str_replace("{format}", "json", $resourcePath);
+
+        
+        // for model (json/xml)
+        if (isset($_tempBody)) {
+            $httpBody = $_tempBody; // $_tempBody is the method argument, if present
+        } elseif (count($formParams) > 0) {
+            $httpBody = $formParams; // for HTTP post (form)
+        }
+        // make the API Call
+        try {
+            list($response, $statusCode, $httpHeader) = $this->apiClient->callApi(
+                $resourcePath,
+                'GET',
+                $queryParams,
+                $httpBody,
+                $headerParams,
+                '\Swagger\Client\ContentEntity\Models\MixedRelatedContent',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\Swagger\Client\ContentEntity\Models\MixedRelatedContent" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)